consensus/types/src/fork_name.rs

```diff
@@ -201,6 +201,46 @@ impl ForkName {
     pub fn gloas_enabled(self) -> bool {
         self >= ForkName::Gloas
     }
+
+    pub fn fork_ascii(self) {
+        if self == ForkName::Fulu {
+            println!(
+                r#"
+                                  ╔═══════════════════════════════════════╗
+                                  ║                                       ║
+                                  ║   TO FULU, MOAR BLOBS TO ETHEREUM     ║
```

Comment:
```suggestion
                                  ║    TO FULU, MOAR BLOBS TO ETHEREUM    ║
```
center, 4 chars left 4 chars right

Reply:
centered